api/graphql/schema.graphql

```diff
@@ -466,10 +448,7 @@ type ScreeningQuestionResponse {
 
 type PoolCandidate {
   id: ID!
-  pool: Pool! @belongsTo @can(ability: "view", resolved: true)
-  poolAdvertisement: PoolAdvertisement
-    @belongsTo(relation: "pool")
-    @can(ability: "viewAdvertisement", resolved: true)
+  pool: Pool! @belongsTo @can(ability: "viewAdvertisement", resolved: true)
```

Comment:
I feel like if we are getting rid of the PoolAdvertisment type, we can also get rid of or rename our "advertisement" permissions and policy functions. Like here, I think `ability: "view"` should work equally well

Reply:
This was addressed as part of [2ff0e2d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6682/commits/2ff0e2d8b691a0c224bcc3bd0b79258d2c7c54ee).